packages/oui-sidebar/README.md

```diff
@@ -0,0 +1,689 @@
+                    <button type="button" class="oui-button oui-button_link oui-button_icon-left">
+                      <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
+                    </button>
+                  </div>
+                  <span class="oui-sidebar-menu__breadcrumb">
+                    Manager › Category 3
+                  </span>
+                  <h3 class="oui-sidebar-menu__title">
+                    Product 1
+                  </h3>
+                </header>
+                <ul class="oui-sidebar-list">
+                  <li class="oui-sidebar-list__item">
+                    <a class="oui-sidebar-link oui-sidebar-link_active" href="">
+                      Service 1
+                    </a>
+                  </li>
+                  <li class="oui-sidebar-list__item">
+                    <a class="oui-sidebar-link" href="">
+                      Service 2
+                    </a>
+                  </li>
+                  <li class="oui-sidebar-list__item">
+                    <a class="oui-sidebar-link" href="">
+                      Service 3
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </li>
+            <li class="oui-sidebar-list__item">
+              <a class="oui-sidebar-link" href="">
+                Product 2
+              </a>
+            </li>
+            <li class="oui-sidebar-list__item">
+              <a class="oui-sidebar-link" href="">
+                Product 3
+              </a>
+            </li>
+          </ul>
+        </div>
+      </li>
+    </ul>
+  </div>
+</nav><nav class="oui-sidebar">
+  <!-- Root -->
+  <div class="oui-sidebar-menu oui-sidebar-menu_fixed">
+    <ul class="oui-sidebar-list">
+      <li class="oui-sidebar-list__item" aria-haspopup="true" aria-expanded="false">
+        <button type="button" class="oui-sidebar-link">
+          Category 1
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item" aria-haspopup="true" aria-expanded="false">
+        <button type="button" class="oui-sidebar-link">
+          Category 2
+        </button>
+      </li>
+      <li class="oui-sidebar-list__item">
+        <button class="oui-sidebar-link oui-sidebar-link_active" aria-haspopup="true" aria-expanded="true">
+          Category 3
+        </button>
+        <!-- Level 1 -->
+        <div class="oui-sidebar-menu oui-sidebar-menu_fixed">
+          <header class="oui-sidebar-menu__header">
+            <div class="oui-sidebar-menu__back">
+              <button type="button" class="oui-button oui-button_link oui-button_icon-left">
+                <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
+              </button>
+            </div>
+            <span class="oui-sidebar-menu__breadcrumb">
+              Manager
+            </span>
+            <h3 class="oui-sidebar-menu__title">
```

Comment:
No `h1` and `h2` above

Reply:
ref: https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks

Reply:
So, this means having a `<h1>` in each `<header>` ?